.github/workflows/deploy-to-azure.yml

```diff
@@ -25,35 +33,47 @@ jobs:
         uses: gradle/gradle-build-action@29c0906b64b8fc82467890bfb7a0a7ef34bda89e # v3
         with:
           arguments: build -x spotlessCheck
-      - name: Login via Azure CLI
-        uses: azure/login@8c334a195cbb38e46038007b304988d888bf676a # v2
+      - name: Login Via Azure CLI
+        uses: azure/login@cb79c773a3cfa27f31f25eb3f677781210c9ce3d # v1
```

Comment:
Why are you reverting to version 1?

Reply:
Thanks Violeta for noticing this.
Can we ignore this for the moment and expect Renovate to make subsequent PRs in order to fix all versions pinning ?

Reply:
I would like to not ignore and not put noise like this in the PR

Reply:
ok, then I'll fix this soon.

Reply:
applied feedback, see https://github.com/reactor/projectreactor.io/pull/150/commits/e0efd7d55c2b2b2592f27e8701b865196964785a

Reply:
there are other dependencies that are not pinned, let me know if you want to fix them too.

Reply:
thanks, now it is obvious that this fallback to version 1 is not intentional!

Reply:
> there are other dependencies that are not pinned, let me know if you want to fix them too.

not is was just this one, when someone is reviewing the PR it is good to have only relevant changes in the PR and no noise.